lib/utils.js

```diff
@@ -690,6 +690,7 @@ export const trackingIdMapping = () => {
     'angdiaryo.org': 'UA-166777432-51',
     'www.spotlightschools.com': 'UA-166777432-37',
     'next-tinynewsdemo': 'UA-166777432-1',
+    atlanticcityfocus: 'UA-166777432-87',
```

Comment:
Curious if there's a reason to use this one over the other one ending in '86'?

Reply:
This is the tracking ID I added to their CMS admin page